internal/storage/elasticsearch/config/config.go

```diff
@@ -235,9 +235,13 @@ func NewClient(c *Configuration, logger *zap.Logger, metricsFactory metrics.Fact
 		After(func(id int64, requests []elastic.BulkableRequest, response *elastic.BulkResponse, err error) {
 			start, ok := m.Load(id)
 			if !ok {
```

Comment:
this bug indicates you don't have a unit test for it
```suggestion
			if ok {
```

Reply:
you are right. Actually I was waiting for the metrics design to be reviewed and approved before writing the unit test as change in logic might affect them.